Update iammeter.markdown

lewei50 opened the pull request:
Add the description of how to use IAMMETER products to monitor the solar pv system in Home Assistant

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

lewei50 committed:
Update iammeter.markdown

Add the description of how to use IAMMETER products to monitor the solar pv system in Home Assistant

frenck commented on source/_integrations/iammeter.markdown:
```diff
@@ -13,12 +13,14 @@ ha_platforms:
 
-`iammeter` provides real-time readings of single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
+`IAMMETER` provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
+[Monitor your solar PV system with IAMMETER energy monitor in Home Assistant](https://community.home-assistant.io/t/monitor-your-solar-pv-system-in-home-assistant-energy-dashboard-vs-energy-management/342710)
```
I don't think we should link to community forum resources from our documentation.

lewei50 replied:
Hi Frenck:
So glad to receive your message again.
I have removed this content temporarily.
But in fact, we want to introduce some specific solutions that use our products in HA here.
Such as how to monitor the solar PV system? how to use the Grafana to display data while using HA to get the data?
Has it permissible to publish such content in the doc?

If so , could we use the content post in our website or the HA official forum?

Thanks for your attention.
BR
Jason

frenck replied:
> Such as how to monitor the solar PV system? how to use the Grafana to display data while using HA to get the data?
> Has it permissible to publish such content in the doc?

We are not the Grafana docs. Things like these can be made available on the community forums or your own resources but are not what we want to have in our end-user documentation.

lewei50 replied:
OK

frenck commented on source/_integrations/iammeter.markdown:
```diff
@@ -13,12 +13,14 @@ ha_platforms:
 
 <p class='img'>
-  <img src='/images/integrations/iammeter/demo.jpg' />
+  ![IAMMETER energy monitor in Home Assistant](https://iammeterglobal.oss-ap-southeast-1.aliyuncs.com/img/image-20211108143148254.png)
+  ![monitor the solar pv system in Home Assistant](https://iammeterglobal.oss-ap-southeast-1.aliyuncs.com/img/image-20211108143245427.png)
```
images need to be hosted from our repository, not external. This is for privacy reasons.

lewei50 replied:
Already done. Thanks

lewei50 committed:
Add files via upload

two pics about solar pv monitoring in Home Assistant

lewei50 committed:
Update iammeter.markdown

upload pics to github respository

frenck commented on source/_integrations/iammeter.markdown:
```diff
@@ -13,12 +13,13 @@ ha_platforms:
 Example Lovelace Dashboard:
 
 <p class='img'>
-  <img src='/images/integrations/iammeter/demo.jpg' />
+<img src='/images/integrations/iammeter/energy-monitor-Home Assistant.png' />
+<img src='/images/integrations/iammeter/solar-pv-monitoring-home-assistant.png' />
 </p>
```
- Please use markdown to include images
- The first filename has spaces, please make it formatted like the second filename
- The first screenshot just shows the Home Assistant energy dashboard, how is that related to this integration?
- The second screenshot is a custom configuration, this is misleading as the end-user might think they will end up getting that when setting up this integration, which is not true.

lewei50 replied:
Please use markdown to include images (**got it**)
The first filename has spaces, please make it formatted like the second filename (**got it**)

The first screenshot just shows the Home Assistant energy dashboard, how is that related to this integration?
The second screenshot is a custom configuration, this is misleading as the end-user might think they will end up getting that when setting up this integration, which is not true.

(**In fact, these two pics are related to this  forum post, https://community.home-assistant.io/t/monitor-your-solar-pv-system-in-home-assistant-energy-dashboard-vs-energy-management/342710
I added this in the first commit and removed it according to your suggestion in the second commit.
Without the introduction of the post, these two pics are indeed easy to mislead the user.
Could I add a section at the last of this doc called "How to monitor your energy consumption" and put these two pics and the forum link there?**)

frenck replied:
If you want to guide users on how to set up energy monitoring with Home Assistant, we should link to that specific page in Home Assistant.

The energy dashboard screenshot is not a feature of this integration, it is a Home Assistant feature. It doesn't belong here.

lewei50 replied:
understood,I have already remove such pics in this doc

frenck commented on source/_integrations/iammeter.markdown:
```diff
@@ -13,12 +13,13 @@ ha_platforms:
 ---
 
-`iammeter` provides real-time readings of single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
+`IAMMETER` provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
```
```suggestion
IAMMETER provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
```

frenck replied:
Why does this link to docs and not a product?

lewei50 replied:
I think this link is like a step-by-step tutorial about how to integrate the IAMMETER products into HA.

frenck replied:
Yes, but the link doesn't suggest that.

lewei50 replied:
Hi, Frenck, 
Thanks for your suggestion.
I have replaced the tutorial page with the official web page of IAMMETER and added two products page.

lewei50 committed:
Update iammeter.markdown

add the products description link.
remove the pics that have no related introduction

lewei50 committed:
Update iammeter.markdown

remove the step-by-step tutorial.

lewei50 commented:
Hi Frenck 
looking forward to your further suggestion.
Thanks

lewei50 commented:
Hi:
So appreciate your attention.
If possible, please let me know whether there are any other questions in this PR.

frenck commented:
Stop calling out messages, please...! Be patient instead.

frenck commented on source/_integrations/iammeter.markdown:
```diff
@@ -11,17 +11,17 @@ ha_codeowners:
+
 
-## Configuration
+# Configuration
```
This change is incorrect

frenck commented on source/_integrations/iammeter.markdown:
```diff
@@ -49,44 +49,44 @@ name:
 {% endconfiguration %}
 
-## Sensors
+# Sensors
```
This change is incorrect, please revert.

frenck reviewed (changes requested):
There are now unused images attached in this PR, please clean that up.

lewei50 committed:
Delete demo.jpg

lewei50 committed:
Delete energy-monitor-Home Assistant.png

lewei50 committed:
Delete solar-pv-monitoring-home-assistant.png

lewei50 committed:
Update iammeter.markdown

lewei50 commented:
> Stop calling out messages, please...! Be patient instead.

Sorry, I am too anxious before.
Already done.
Please check it at your convenience.
Thanks.

frenck commented on source/_integrations/iammeter.markdown:
```diff
@@ -11,15 +11,14 @@ ha_codeowners:
   - '@lewei50'
 ha_platforms:
   - sensor
+
+
```
```suggestion
```

frenck committed:
Tweak
